File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelReferencedInSpatialStructure/IfcRelReferencedInSpatialStructure.vb
Public Class IfcRelReferencedInSpatialStructure Inherits IfcRelConnects
    Public RelatedElements As SET [1:?] OF IfcSpatialReferenceSelect
    Public RelatingStructure As IfcSpatialElement

    ' === WHERE CLAUSES ===
    ' AllowedRelatedElements : SIZEOF(QUERY(temp <* RelatedElements | ('IFC4X3_DEV_73740fe4.IFCSPATIALSTRUCTUREELEMENT' IN TYPEOF(temp)) AND (NOT ('IFC4X3_DEV_73740fe4.IFCSPACE' IN TYPEOF(temp))) )) = 0
End Class

File: src/main/resources/output/IFC4X3_java/Entities/IfcRoot/IfcRelationship/IfcRelConnects/IfcRelReferencedInSpatialStructure/IfcRelReferencedInSpatialStructure.java
public class IfcRelReferencedInSpatialStructure extends IfcRelConnects {
    public SET [1:?] OF IfcSpatialReferenceSelect RelatedElements;
    public IfcSpatialElement RelatingStructure;

    // === WHERE CLAUSES ===
    // AllowedRelatedElements : SIZEOF(QUERY(temp <* RelatedElements | ('IFC4X3_DEV_73740fe4.IFCSPATIALSTRUCTUREELEMENT' IN TYPEOF(temp)) AND (NOT ('IFC4X3_DEV_73740fe4.IFCSPACE' IN TYPEOF(temp))) )) = 0
}
